src/http_client.js

```diff
@@ -11,7 +11,9 @@ const {
 const {
     isNode,
 } = require('./utils');
-const { version } = require('../package.json');
+
+// eslint-disable-next-line no-eval
+const { version } = eval(`require('../package.json')`);
```

Comment:
Webpack... I guess??

Reply:
tsc actually, since otherwise it tends to clone the package.json. It's ugly, I know, and I'll see tomorrow what I can do to solve the need for this (since it might also make webpack angry)

Reply:
Okay. Not sure if this is still relevant but https://twitter.com/kamilogorek/status/1102272038411137025

Reply:
That's very insightful, thanks! It's actually gonna be weird..In webpack I need to let it do it's thing, in TS i need it to just require as is, without importing... I'm sure I'll figure something out 👀

Reply:
Was quite a headache, but I figured out how to make both worlds happy 🎉